src/Domain/RegularGivingMandate.php

```diff
@@ -37,7 +37,7 @@ class RegularGivingMandate extends SalesforceWriteProxy
     private readonly UuidInterface $uuid;
 
     #[ORM\Embedded(columnPrefix: 'person')]
-    public PersonId $donorId;
+    private PersonId $donorId;
```

Comment:
Unless I found an exception previously that I now can't remember, then all Doctrine properties should be private (or at least not public) to allow lazy-loading to work.